src/components/movies-list/movies-list.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import VisibilityFilterInput from '../visibility-filter-input/visibility-filter-input';
+import { MovieCard } from '../movie-card/movie-card';
+
+const mapStateToProps = state => {
+  const { visibilityFilter } = state;
+  return { visibilityFilter };
+};
+
+function MoviesList(props) {
+  const { movies, visibilityFilter } = props;
+  let filteredMovies = movies;
+
+  if (visibilityFilter !== '') {
+    filteredMovies = movies.filter(m => m.Title.includes(visibilityFilter));
+  }
+
+  if (!movies) return <div className="main-view"/>;
+
+  return <div className="movies-list">
+    <VisibilityFilterInput visibilityFilter={visibilityFilter} />
+    {filteredMovies.map(m => <MovieCard key={m._id} movie={m}/>)}
+  </div>;
+}
+
+export default connect(mapStateToProps)(MoviesList);
```

Comment:
this error is due to 

> no newline at the end of file

There is a setting in your editor to add a newline to the end of each file on save. Not too important just now, just an FYI